src/schemas.ts

```diff
@@ -36,3 +36,180 @@ export const unregisteredAddressesSchema = {
 	},
 	required: ['unregisteredAddresses'],
 };
+
+export const accountSchema = {
```

Comment:
Could you please confirm if the schemas are taken exactly as is from Lisk Core v3/Lisk SDK v5, as applicable?